crates/cxx-qt-lib-headers/include/qcolor.h

```diff
@@ -0,0 +1,34 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#pragma once
+
+#include <QColor>
+
+#include <memory>
+
+#include "qt_types.h"
+
+template<>
+struct rust::IsRelocatable<QColor> : std::true_type
+{
+};
+static_assert(QTypeInfo<QColor>::isRelocatable);
```

Comment:
does the assert need to be in the header? or could this also be in the source? if we are trying to avoid multiple static_asserts at compile time?

Then just leave the template here ?

Reply:
We could also just remove this static_assert, as it's kind of pointless to check something we're literally defining to be true in the line above :thinking:

Reply:
The point of the check is to check that Qt has defined that it is relocatable. The line above is telling Rust it is relocatable. So we still need the static_assert.

As what happens if suddenly QColor is not relocatable and we marked it wrong?

Reply:
Ooops, you're right, I misread, will move these into the CPP files then.